### DISCUSSION TITLE
i3 and Hyperland

### DISCUSSION THREAD

**Discussion by runetech0** [i3/i3#6252] [Q&A] [answered]
I have been a user of i3 for almost 3 years now and very happy with that. 
I love controlling everything with my keyboard so that never lets me leave i3 and think about anything else. 

But recently I looked into hyperland and thinking to move to hyperland because of the beauty it has and animations etc.
But I'm not sure if it'll let me control everything using the keyboard as much as you can with i3. 
And how's the support for NVidia GPUs? 

If anyone have tried hyperland here I would like to know how does it feels and is it worth it? 

Also, what distro are you using the hyperland on?

**Comment by okraits** [comment, marked as answer]
Shouldn't you better ask this question in some hyprland community?
Most people reading this are using i3 and the majority might never ever have tried hyprland.

i3 is a well designed and built, very stable wm. Thanks to the lack of animations its responses and actions are very fast and without visual distractions.

hyprland, on the other hand, is a wayland compositor. Which means that if you're using it, you are also using a different display server implementation. Which means that you might need different programs for some usecases and you need to configure some things differently, compared to using i3 and the X Window system.

**Reply by runetech0** [reply]
Thanks! I was actually gonna post this in r/i3wm but then I saw that's inactive so I posted here.
I'm a user of i3 so I know it's awesome but I was just wondering if the Hyprland is kind of super set of i3 but seems like a different case.
